iocore/net/P_SSLNetVConnection.h

```diff
@@ -161,6 +162,15 @@ class SSLNetVConnection : public UnixNetVConnection
     return npnEndpoint;
   }
 
+  void disableProtocol(int idx);
+  void enableProtocol(int idx);
+
+  void
+  setEnabledProtocols(const SessionProtocolSet &protos)
```

Comment:
It might make more sense if it was `setSupportedProtocols` and the variable name was `proto_supported`. It was hard to understand `protoenabled` contains protocols disabled and those are just marked as disabled.

Reply:
The npnset contains all the protocols.  The protocolenabled is just a bit array that marks the protocols that should be enabled.  So I think that enabled is the more appropriate name than supported.  Could rename the npnset variable to be some variant of supported protocols.